django-verdant/rca/models.py

```diff
@@ -1319,9 +1365,9 @@ class NewsItem(Page, SocialFields):
 
     def get_related_news(self, count=4):
         return NewsItem.get_related(
-            areas=list(self.areas.values_list('area', flat=True)),
-            programmes=list(self.related_programmes.values_list('programme', flat=True)),
-            schools=list(self.related_schools.values_list('school', flat=True)),
+            areas=Area.objects.filter(id__in=self.areas.values_list('area_id', flat=True)),
```

Comment:
I don't think that it's so important on this project, but It seem to me that it's better to use `pk` alias instead of `id`. It will hard to replace all `id`s with `pk`s in code, if primary key field will be renamed.

Also `flat=True` is not necessary here, so you can write something like this.
```python
Area.objects.filter(pk__in=self.areas.values_list('area')
```